examples/manually-instrumented-chatbot/frontend/Makefile

```diff
@@ -1,5 +1,5 @@
-server:
-	streamlit run app.py
+frontend:
+	streamlit run ./src/app.py
```

Comment:
renamed this to src since frontend/frontend seemed weird, don't know about streamlit conventions though